.eslintrc.js

```diff
@@ -25,7 +25,7 @@ const tslintRules = {
   ],
   "hex-format": true,
   "import-spacing": true,
-  "interface-sort-keys": true,
+  "interface-sort-keys": false,
```

Comment:
Actually `member-ordering` already has interface sort feature. Using `interface-sort-keys` along with `member-ordering` works great until you want to add methods to an interface.

For example
```typescript
interface ISomething {
   xxxField: string;
   yyyField: string;
   aaaMethod(): string; // interface-sort-keys won't like this.
}
```
and
```typescript
interface ISomething {
   aaaMethod(): string; // member ordering won't like this.
   xxxField: string;
   yyyField: string;
}
```